src/System.Runtime.Analyzers/Core/SpecifyCultureInfo.cs

```diff
@@ -24,15 +27,86 @@ public abstract class SpecifyCultureInfoAnalyzer : DiagnosticAnalyzer
                                                                              s_localizableMessage,
                                                                              DiagnosticCategory.Globalization,
                                                                              DiagnosticSeverity.Warning,
-                                                                             isEnabledByDefault: false,
+                                                                             isEnabledByDefault: true,
                                                                              description: s_localizableDescription,
-                                                                             helpLinkUri: null,     // TODO: add MSDN url
+                                                                             helpLinkUri: Uri,
                                                                              customTags: WellKnownDiagnosticTags.Telemetry);
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(Rule);
 
         public override void Initialize(AnalysisContext analysisContext)
         {
+            analysisContext.RegisterCompilationStartAction(csaContext =>
+            {
+                var cultureInfoType = csaContext.Compilation.GetTypeByMetadataName("System.Globalization.CultureInfo");
+                if (cultureInfoType != null)
+                {
+                    csaContext.RegisterOperationAction(oaContext =>
+                    {
+                        var invocationExpression = (IInvocationExpression)oaContext.Operation;
+                        var targetMethod = invocationExpression.TargetMethod;
+                        if (targetMethod.ContainingType == null)
+                        {
+                            return;
+                        }
+
+                        var methodsWithSameNameAsTargetMethod = targetMethod.ContainingType.GetMembers(targetMethod.Name).OfType<IMethodSymbol>();
+                        if (methodsWithSameNameAsTargetMethod.Count() < 2)
+                        {
+                            return;
+                        }
+
+                        var correctOverloads = methodsWithSameNameAsTargetMethod.Where(candidateMethod =>
+                        {
+                            if (targetMethod.Parameters.Count() + 1 != candidateMethod.Parameters.Count())
+                            {
+                                return false;
+                            }
+
+                            // Assume CultureInfo is the last parameter so j, which is the index of the parameter
```

Comment:
And in that case, IMO the code would be clearer if your had a variable `offset` that was set to 0 or 1, and then your comparison would be

```
targetMethod.Parameters[i].Type.Equals(candidateMethod.Parameters[i + offset].Type
```

... instead of maintaining two loop variables.